xcp_d/utils/bids.py

```diff
@@ -441,9 +475,9 @@ def get_preproc_pipeline_info(input_type, fmri_dir):
     if os.path.isfile(dataset_description):
         with open(dataset_description) as f:
             dataset_dict = json.load(f)
-
+    try:
         info_dict["version"] = dataset_dict["GeneratedBy"][0]["Version"]
-    else:
+    except:
```

Comment:
I try to avoid try/except statements if possible. What does the HCP dataset description file look like?

Reply:
Does it have a "GeneratedBy" field?

Reply:
Yes